docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| CloseTime                  | Datetime | The time at which the Workflow Execution completed.                                                                                                                          |
+| ExecutionTime              | Datetime | Same as StartTime for the most cases but different for cron Workflows and retried Workflows. For them it is the time at which the Workflow Execution actually begin running. |
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
+| TemporalChangeVersion      | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
+| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                               |
+| BatcherNamespace           | Keyword  | Used by internal batcher to indicate the Namespace where batch operation was applied to.                                                                                     |
+| BatcherUser                | Keyword  | Used by internal batcher to indicate the user who started the batch operation.                                                                                               |
+| NamespaceId                | Keyword  | Identifies the Namespace of the Search Attribute.                                                                                                                            |
+| TemporalScheduledStartTime | Datetime | The time that the Workflow is schedule to start at.                                                                                                                          |
```

Comment:
@dnr to check, but seems okay to me

Reply:
i assume we do not want TemporalNamespaceDivision documented here. :) OSS should confirm that too

Reply:
You might say something like, "This attribute will be set on workflows started by a Schedule. It is the time that the workflow is scheduled to start according to the Schedule spec (or was manually triggered)."

Reply:
I think it's fine to document TemporalNamespaceDivision. It's not a secret.. they can see their batch and schedule workflows if they want. We talked about letting users use that search attribute as well, though no movement in that direction yet. However, they should keep in mind that the batch and schedules APIs make no guarantees about how they are implemented, specifically w.r.t workflows.

Reply:
`TemporalNamespaceDivision` is hiddden. Same as `NamespaceId`. Don't document them.